src/Config_Command.php

```diff
@@ -153,16 +153,14 @@ public function create( $_, $assoc_args ) {
 			WP_CLI::error( '--dbprefix can only contain numbers, letters, and underscores.' );
 		}
 
-		$mysql_db_connection_args = [
-			'execute' => ';',
-			'host'    => $assoc_args['dbhost'],
-			'user'    => $assoc_args['dbuser'],
-			'pass'    => $assoc_args['dbpass'],
-		];
-
 		// Check DB connection.
 		if ( ! Utils\get_flag_value( $assoc_args, 'skip-check' ) ) {
-			Utils\run_mysql_command( '/usr/bin/env mysql --no-defaults', $mysql_db_connection_args );
+			// phpcs:disable WordPress.DB.RestrictedFunctions,WordPress.PHP.NoSilencedErrors.Discouraged
```

Comment:
it would be awesome here if we provided a description in a comment explaining why the command is calling a restricted function, similar to what's in the PR description.